core/audits/bootup-time.js

```diff
@@ -62,6 +66,33 @@ class BootupTime extends Audit {
     };
   }
 
+  /**
+   * @param {LH.Artifacts} artifacts
+   * @param {LH.Audit.Context} context
+   * @return {Promise<number>}
+   */
+  static async getTbtImpact(artifacts, context) {
+    let tbtImpact = 0;
+
+    try {
+      const metricComputationData = Audit.makeMetricComputationDataInput(artifacts, context);
+      const tasks = await TBTImpactTasks.request(metricComputationData, context);
```

Comment:
In theory it would be better to compute these tasks once and use them for normal procedure on this audit. Unfortunately this would break one of our test cases which asserts this audit get's no data for a bad trace.

I'm open to changing that logic, it means this audit would error whenever TBT errors which seems reasonable to me. It didn't seem important enough just to make the code neater IMO.